src/app/app.constants.ts

```diff
@@ -60,12 +60,15 @@ const userNav: UserNavModel = {
 
 const regOrgTitle: AppTitlesModel = {
   name: 'Register to manage civil, family, and tribunal law cases',
-  url: '/register-org/register/'
+  url: '/register-org/register/',
+  hideBranding: true
 };
 
 const manageOrgTitle: AppTitlesModel = {
   name: 'Manage organisation',
-  url: '/'
+  url: '/',
+  hideBranding: true
```

Comment:
Do you need it on this one? I thought you only wanted to hide the branding for "Register org"...?